Docs/RHCL-RELEASE-GUIDE.md

```diff
@@ -0,0 +1,873 @@
+   - Workspace: `api-management-tenant`
+   - Managed workspace view access: `rhtap-releng-tenant`
+
+2. **OpenShift CLI (`oc`)**
+   - Install: https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html
+   - Login to Konflux cluster
+
+3. **GitLab Access**
+   - Access to `releng/konflux-release-data` repository
+
+### Configuration Files
+4. **ReleasePlanAdmissions (RPAs)**
+   - Located in [`releng/konflux-release-data/config/stone-prd-rh01.pg1f.p1/product/ReleasePlanAdmission/api-management/`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/tree/main/config/stone-prd-rh01.pg1f.p1/product/ReleasePlanAdmission/api-management)
+   - Files follow naming pattern: `rhcl-{version}-{component}-{environment}-rhel9.yaml`
+   - Examples:
+     - [`rhcl-1-2-rhcl-operator-prod-rhel9.yaml`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/ReleasePlanAdmission/api-management/rhcl-1-2-rhcl-operator-prod-rhel9.yaml) (production)
+     - [`rhcl-1-2-authorino-stage-rhel9.yaml`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/ReleasePlanAdmission/api-management/rhcl-1-2-authorino-stage-rhel9.yaml) (staging)
+     - [`rhcl-file-based-catalog-stage.yaml`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/ReleasePlanAdmission/api-management/rhcl-file-based-catalog-stage.yaml) (FBC staging)
+
+5. **Enterprise Contract Policies**
+   - Standard policies: [`releng/konflux-release-data/config/stone-prd-rh01.pg1f.p1/product/EnterpriseContractPolicy/`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/tree/main/config/stone-prd-rh01.pg1f.p1/product/EnterpriseContractPolicy)
+   - RHCL custom build policy: [`config/policy.yaml`](https://github.com/rh-api-management/rh-connectivity-link/blob/alpha/config/policy.yaml) in rh-api-management repo
+     - Used during build/test phase (developer workspace)
+     - Defines allowed registries, temporary exemptions for CVE/RPM checks
+   - RHCL release policies (referenced in RPA `spec.policy` field):
+     - [`registry-api-management-prod`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/stone-prd-rh01.pg1f.p1/product/EnterpriseContractPolicy/registry-api-management-prod.yaml) (production component releases)
+     - [`fbc-standard`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/config/common/product/EnterpriseContractPolicy/fbc-standard.yaml) (file-based catalog releases)
+
+6. **Constraints**
+   - Located in: [`releng/konflux-release-data/constraints/product/api-management.yaml`](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/constraints/product/api-management.yaml)
+   - Validates RPA values (registries, tags, policies)
+
+7. **Pyxis Delivery Repositories**
```

Comment:
We don't use Pyxis directly, we use Comet which makes the changes in Pyxis for us

There is also the cicada tool which does the same thing but we haven't migrated to that yet